plugins/utils/ccc.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "ccc";
+  originalName = "ccc.nvim";
+  defaultPackage = pkgs.vimPlugins.ccc-nvim;
+
+  maintainers = [helpers.maintainers.JanKremer];
+
+  settingsOptions = {
+    default_color = helpers.defaultNullOpts.mkStr "#000000" ''
+      The default color used when a color cannot be picked. It must be HEX format.
```

Comment:
Should there be a warning/assertion that this is a valid hex string?

Reply:
I don't think we ever check the validity of strings (apart from enumerations)